apps/roam/src/components/DiscourseContextOverlay.tsx

```diff
@@ -27,6 +28,42 @@ import getAllPageNames from "roamjs-components/queries/getAllPageNames";
 import { Result } from "roamjs-components/types/query-builder";
 import createBlock from "roamjs-components/writes/createBlock";
 import { getBlockUidFromTarget } from "roamjs-components/dom";
+import {
```

Comment:
It is unclear to me what the use case is of importing these functions just to pass them back to the same util file.